packages/beacon-node/test/spec/presets/light_client/sync.ts

```diff
@@ -124,7 +124,7 @@ export const sync: TestRunnerFn<SyncTestCase, void> = (fork) => {
       for (const [i, step] of testcase.steps.entries()) {
         try {
           if (isProcessUpdateStep(step)) {
-            const currentSlot = Number(step.process_update.current_slot as bigint);
```

Comment:
This was intended in case we change the type for `current_slot` because `Number` accepts any type. If anyone have idea about it's workaround do share.